File: 프로그래머스/lv0/120830. 양꼬치/양꼬치.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n, int k) {
    int answer = 0;
    double num = n;
    if(n >= 10) {
        k -= n/10;
    }
    answer = n * 12000 + k * 2000;
    return answer;
}

File: 프로그래머스/lv0/120830. 양꼬치/양꼬치.js
function solution(n, k) {
    var answer = 0;
    // 양꼬치 : n * 12000
    // 음료수 : k * 2000
    let yang = n * 12000;
    let drink = k * 2000;
    
    // 할인가격 : n / 10 것의 몫
    let discount = Math.floor(n/10) * 2000;
    
    return yang + drink - discount;
}